backend/scripts/wmg_query_tiledb_api_examples.py

```diff
@@ -0,0 +1,39 @@
+# Examples of ad hoc queries of the WMG Expression Summary cube using the TileDB query API.
+
+import os
+
+import pandas as pd
+
+from backend.wmg.data.snapshot import _open_cube, EXPRESSION_SUMMARY_CUBE_NAME
+
+pd.set_option('max_columns', 10)
+pd.set_option('display.width', 256)
+
+os.environ['DEPLOYMENT_STAGE'] = 'dev'
+
+if __name__ == '__main__':
+    cube = _open_cube(f's3://cellxgene-wmg-dev/1662103227/{EXPRESSION_SUMMARY_CUBE_NAME}/')
```

Comment:
as @atarashansky mentioned above, make the bucket name dynamic based on the DEPLOYMENT_STAGE